Quadriga/src/main/java/edu/asu/spring/quadriga/web/resolver/TestResolverController.java

```diff
@@ -0,0 +1,42 @@
+package edu.asu.spring.quadriga.web.resolver;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.spring.quadriga.domain.resolver.IProjectHandleResolver;
+import edu.asu.spring.quadriga.domain.resolver.Status;
+import edu.asu.spring.quadriga.service.resolver.IProjectHandleResolverManager;
+
+@Controller
+public class TestResolverController {
+
+    @Autowired
+    private IProjectHandleResolverManager resolverManager;
+
+    @RequestMapping(value = "/auth/resolvers/test", method = RequestMethod.POST)
+    public String editPage(Principal principal, Model model, @RequestParam("resolverId") String resolverId,
+            RedirectAttributes redirectAttributes) {
+
+        IProjectHandleResolver projectHandleResolver = resolverManager.getProjectHandleResolver(resolverId);
+
+        projectHandleResolver.setValidation(resolverManager.getValidationProjectHandleResolver(projectHandleResolver));
+
+        if (projectHandleResolver.getValidation() == Status.PASSED) {
+            redirectAttributes.addFlashAttribute("show_success_alert", true);
+            redirectAttributes.addFlashAttribute("success_alert_msg", "Test on Resolver successful.");
```

Comment:
The success message should be in the messages files, check others controllers how to do that. Also, message should be: "Regex pattern successfully applied to handle examples."